AgileBoard.Api/src/AgileBoard.Api/Enums/StoryPoints.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace AgileBoard.Api.Enums
+{
+    public enum StoryPoints
```

Comment:
Story point should not be enum, it should be an object and if you are using Fibonnacci scale, then you can easly write validation for that and unit test it :)

Reply:
removed StoryPoints enum. Will add the validation later